ee/control/control.go

```diff
@@ -278,6 +278,14 @@ func (cs *ControlService) RegisterSubscriber(subsystem string, subscriber subscr
 	cs.subscribers[subsystem] = append(cs.subscribers[subsystem], subscriber)
 }
 
+func (cs *ControlService) Message(method string, params interface{}) error {
+	if cs.fetcher == nil {
```

Comment:
When will this be nil? If it's just in testing, the error seems fine.

But if it's a startup sequencing thing, should we async in here? (Downside is that it's a lot of complexity, and maybe the caller can use backoff if it matters)

Reply:
it will never be nli, removed nil check